examples/terraform/openstack/variables.tf

```diff
@@ -88,6 +88,12 @@ variable "control_plane_flavor" {
   type        = string
 }
 
+variable "control_plane_group_policy" {
+  description = "OpenStack instance group policy for the control plane nodes. Cannot be changed without recreating VMs!"
+  default     = "soft-anti-affinity"
```

Comment:
Can we somewhere document supported values and why we recommend `soft-anti-affinity`?

Reply:
There are four policies supported.

From https://docs.openstack.org/nova/2024.2/user/server-groups.html:

* `affinity`
* `anti-affinity`
* `soft-affinity`
* `soft-anti-affinity`

`soft-anti-affinity` looks like a good default:

> Attempts to restrict instances belonging to the server group to separate hosts. Where it is not possible to schedule all instances to separate hosts, they will be scheduled on as many separate hosts as possible.